CHANGELOG.md

```diff
@@ -14,7 +14,9 @@ _None_
 
 ### Bug Fixes
 
-_None_
+* .swift-version: fix .swift-version with Swift 5.0 to build in 6.2.0 or later.  
+  [@cfiken](https://github.com/cfiken)
+  [#764](https://github.com/SwiftGen/SwiftGen/issue/764)
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/SwiftGen/SwiftGen/blob/481a130b0d621c5899aecbb375e204b1d77d1b0c/CHANGELOG.md#L19"></span>This looks like a GitHub link URL with a typo. Issue links should use <code>/issues/123</code> (plural) and PR links should use <code>/pull/123</code> (singular).</td>
    </tr>
  </tbody>
</table>